features/polyExplorer/src/context/explorer-context.jsx

```diff
@@ -83,6 +83,8 @@ const loadStoriesMetadata = () => {
     };
 };
 
+const intermediateScreens = ["/search", "/entity-details"];
```

Comment:
Naming nit: I suppose what we're skipping here are specific _routes_, techincally, not _screens_, since we can't be sure which routes lead to which concrete _screen_. So I'd suggest `intermediateRoutes`, or even `routesToSkipOnBack`, or something like this :D Would probably move it into `handleBack` too, since it's only needed there. But not important.

Reply:
No I like it, far more descriptive